workos/user_management.py

```diff
@@ -300,6 +307,7 @@ def list_organization_memberships(
         Kwargs:
             user_id (str): Filter Organization Memberships by user. (Optional)
             organization_id (str): Filter Organization Memberships by organization. (Optional)
+            statuses (list): Filter Organization Memberships by status. (Optional)
```

Comment:
do we generally use this naming pattern for list params?

Reply:
Do mean the pluralized parameter name? Seems like we do in the Python SDK:
https://github.com/workos/workos-python/blob/main/workos/organizations.py#L41
https://github.com/workos/workos-python/blob/main/workos/events.py#L38

Reply:
no strong opinion then but I feel `statuses` seems a bit weird. Maybe `status_list`

Reply:
Down to change it. That being said `statuses` is how it's defined in the API/other SDKs. Maybe it's better to be consistent, thoughts?

Reply:
I think we should be consistent and stick with `statuses`.